main/src/components/Apps/AppPanel.vue

```diff
@@ -248,12 +248,22 @@
 							</b-dropdown>
 						</div>-->
 						<!-- Sort End -->
+
+						<div class="is-flex-grow-1 is-flex is-justify-content-flex-end">
+							<b-input type="text" 
+									@input="debounceSearchInput" 
```

Comment:
Opted to using a `debounced` input event listener rather than `v-model` due to noticeable sluggish-ness when using `v-model`